apps/web/composables/useOrderAgain/useOrderAgain.ts

```diff
@@ -97,11 +97,12 @@ export const useOrderAgain: UseOrderAgainReturn = () => {
         orderItem.itemVariationId > 0 &&
         state.value.order &&
         orderGetters.isItemSalableAndActive(state.value.order, orderItem) &&
-        orderGetters.hasAllOrderPropertiesAvailable(orderItem)
+        orderGetters.hasAllOrderPropertiesAvailable(orderItem) &&
+        orderGetters.getOrderAgainOrderItemStock(orderItem) > 0
       ) {
         items.push({
           productId: orderItem.itemVariationId,
-          quantity: orderItem.quantity,
+          quantity: orderGetters.getOrderAgainOrderItemStock(orderItem),
```

Comment:
is that right? we want the same quantity as the order, only if the quantity is not available we want to push the available stock.

Reply:
The stock from the backend is "Real available stock" > "Ordered quantity" => "Ordered Quantity"
and if "Ordered Quantity" > "Real available stock" => "Real available stock"